coreapps-web-2.2/src/main/java/org/openmrs/module/coreapps/diagnosisCompatibility/DiagnosisServiceCompatibility2_2.java

```diff
@@ -0,0 +1,23 @@
+package org.openmrs.module.coreapps.diagnosisCompatibility;
+
+import java.util.List;
+
+import org.openmrs.Diagnosis;
+import org.openmrs.Encounter;
+import org.openmrs.api.DiagnosisService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+public class DiagnosisServiceCompatibility2_2 implements DiagnosisServiceCompatibility  {
+	
+	@Autowired
+	@Qualifier("diagnosisService")
+	DiagnosisService diagnosisService;
+	
+	@Override
+	public List<Diagnosis> getDiagnoses(Encounter encounter) throws Exception {
+		List<Diagnosis> list = diagnosisService.getDiagnoses(encounter);
```

Comment:
But the method your trying to acces doesnt exist at the moment , it still lives in your [PR](https://github.com/openmrs/openmrs-core/pull/3682/files)

Reply:
> But the method your trying to acces doesnt exist at the moment , it still lives in your [PR](https://github.com/openmrs/openmrs-core/pull/3682/files)

Thats true but since this method exists locally,i am expecting my tests to pass locally

Reply:
cool